apps/dashboard/app/views/layouts/application.html.erb

```diff
@@ -25,44 +25,41 @@
 </head>
 <body>
   <header <%= 'data-turbolinks-permanent' if Configuration.turbolinks_enabled? %>>
-    <nav class="navbar navbar-<%= ENV['OOD_NAVBAR_TYPE'].present? ? ENV['OOD_NAVBAR_TYPE'] : 'inverse' %> navbar-static-top">
-    <div class="container-fluid">
-      <div class="navbar-header">
-        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-9" aria-expanded="false"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
-        <% if ENV['OOD_DASHBOARD_HEADER_IMG_LOGO'].present? %>
-          <a class="navbar-brand navbar-brand-logo" href="<%= root_path %>"><img src="<%= ENV['OOD_DASHBOARD_HEADER_IMG_LOGO'] %>"></a>
-        <% else %>
-          <a class="navbar-brand" href="<%= root_path %>"><%= OodAppkit.dashboard.title.html_safe %></a>
-        <% end %>
-      </div>
-      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-9">
-        <ul class="nav navbar-nav">
-          <%= render partial: 'layouts/nav/featured_apps', locals: { group: @featured_group } unless @pinned_apps.empty? %>
```

Comment:
this partial needs to be included.

Reply:
Good catch!

Reply:
Fixed in https://github.com/OSC/ondemand/pull/927/commits/a669c0d38c8594c9c768d29ec5d74856e9c57277